### PR TITLE
Added city card info overlay when tapping on annotation

### PR DESCRIPTION
**Pull request by matt-martindale**
When Users search for a city and tap on the annotation, a card overlay pops up to hold the city info

### CODE REVIEW

File: labs-ios-starter/Map/PresentationController.swift [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  
+  override func containerViewWillLayoutSubviews() {
+      super.containerViewWillLayoutSubviews()
+    presentedView!.roundCorners([.topLeft, .topRight], radius: 22)
+  }
+
+  override func containerViewDidLayoutSubviews() {
+      super.containerViewDidLayoutSubviews()
+      presentedView?.frame = frameOfPresentedViewInContainerView
+      blurEffectView.frame = containerView!.bounds
```
Comment (clc80) [inline labs-ios-starter/Map/PresentationController.swift]: There's several places where you use ! Might there be a way to remove some?

File: labs-ios-starter/View/OverlayView.swift [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        // Not allowing the user to drag the view upward
+        guard translation.y >= 0 else { return }
+        
+        // setting x as 0 because we don't want users to move the frame side ways!! Only want straight up or down
+        view.frame.origin = CGPoint(x: 0, y: self.pointOrigin!.y + translation.y)
+        
+        if sender.state == .ended {
+            let dragVelocity = sender.velocity(in: view)
+            if dragVelocity.y >= 1300 {
+                self.dismiss(animated: true, completion: nil)
```
Comment (clc80) [inline labs-ios-starter/View/OverlayView.swift]: if the completion is nil you can leave this out of the code.

**Review by clc80 — APPROVED** [review]
Great job Matt. Easy to read code, can see that you used some blur effects to make the card stand out.

**Review by ghost — APPROVED** [review]
Very well organized and clean code, nice job

**Pull request merged**